ee/clickhouse/models/event.py

```diff
@@ -80,9 +80,13 @@ def get_id(self, event):
 
     def get_properties(self, event):
         if len(event) >= 10 and event[8] and event[9]:
-            return dict(zip(event[8], event[9]))
+            prop_vals = [_unpad_property(res) for res in event[9]]
+            return dict(zip(event[8], prop_vals))
         else:
-            return json.loads(event[2])
+            props = json.loads(event[2])
+            for key, value in props.items():
+                props.update({key: _unpad_property(value)})
+            return props
```

Comment:
Nitpick: Can be shortened to `{ key: _unpad_property(value) for key, value in props.items() }`

This should be slightly faster than updating dict key by key.

Reply:
got it thanks!